IndicoToolkit/AutoReview/AutoReviewFunctions.cs

```diff
@@ -0,0 +1,225 @@
+        /// Accepts predictions above a given confidence threshold
+        /// </summary>
+        /// <param name="predictions">List of predictions</param>
+        /// <param name="labels">List of labels</param>
+        /// <param name="confThreshold">Threshold to accept predictions</param>
+        /// <returns>
+        /// all predictions 
+        /// </returns>
+        public List<Prediction> acceptByConfidence(List<Prediction> predictions, List<string> labels = default, float confThreshold = .98f)
+        {
+            foreach (Prediction prediction in predictions)
+            {
+                if (prediction.getValue("rejected") == null)
+                {
+                    if (!emptyOrMatchingLabel(prediction, labels))
+                    {
+                        continue;
+                    }
+                    string predLabel = prediction.getLabel();
+                    if (prediction.getValue($"confidence.{predLabel}") > confThreshold)
+                    {
+                        prediction.setValue("accepted", true);
+                    }
+                }
+            }
+            return predictions;
+        }
+
+        /// <summary>
+        /// Accepts all predictions for a class if all their values are the same, and all confidence scores are above a given confidence threshold
+        /// </summary>
+        /// <param name="predictions">List of predictions</param>
+        /// <param name="labels">List of labels</param>
+        /// <param name="confThreshold">Threshold to accept predictions</param>
+        /// <returns>
+        /// all predictions 
+        /// </returns>
+        public List<Prediction> acceptByAllMatchAndConfidence(List<Prediction> predictions, List<string> labels = default, float confThreshold = .98f)
+        {
+            Dictionary<string, HashSet<string>> predictionMap = new Dictionary<string, HashSet<string>>();
+            foreach (Prediction prediction in predictions)
+            {
+                if (prediction.getValue("rejected") == null)
+                {
+                    if (!emptyOrMatchingLabel(prediction, labels))
+                    {
+                        continue;
+                    }
+                    string predLabel = prediction.getLabel();
+                    if (prediction.getValue($"confidence.{predLabel}") > confThreshold)
+                    {
+                        if (predictionMap.TryGetValue(predLabel, out var temp))
+                        {
+                            predictionMap[predLabel].Add((string)prediction.getValue("text"));
+                        }
+                        else
+                        {
+                            predictionMap[predLabel] = new HashSet<string>() { (string)prediction.getValue("text") };
+                        }
+                    }
+                    else
+                    {
+                        predictionMap[predLabel] = new HashSet<string>() { "~", "~~" };
```

Comment:
I think intention here is to just make sure the Count of predictionMap[predLabel] will not be one. Can reconsider how we are approaching this problem as a whole if we think this is too hacky.